<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QT3DReconstructionClass</class>
 <widget class="QMainWindow" name="QT3DReconstructionClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1383</width>
    <height>833</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>J3DGUI</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>J3D_GUI.ico</normaloff>J3D_GUI.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>10</y>
      <width>521</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>输出日志</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_engine">
    <property name="geometry">
     <rect>
      <x>1080</x>
      <y>10</y>
      <width>291</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>Engine</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>750</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>当前程序运行状况：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_process">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>750</y>
      <width>391</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>等待任务</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>760</y>
      <width>791</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="mvsviewer" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>-150</x>
      <y>-190</y>
      <width>1361</width>
      <height>661</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1383</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>分步三维重建</string>
    </property>
    <addaction name="actionMatchFeature"/>
    <addaction name="actionSFM"/>
    <addaction name="separator"/>
    <addaction name="actionDenseCloud"/>
    <addaction name="action_reconstrctMesh"/>
    <addaction name="action"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actionopen_mvs_file"/>
    <addaction name="action_addSensorWidth"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>全自动三维重建</string>
    </property>
    <addaction name="action_fullauto"/>
   </widget>
   <addaction name="menu_3"/>
   <addaction name="menu_4"/>
   <addaction name="menu_2"/>
  </widget>
  <action name="actionMatchFeature">
   <property name="text">
    <string>特征匹配</string>
   </property>
  </action>
  <action name="actionSFM">
   <property name="text">
    <string>SFM及稀疏点云生成</string>
   </property>
  </action>
  <action name="actionDenseCloud">
   <property name="text">
    <string>密集点云生成</string>
   </property>
  </action>
  <action name="action_addSensorWidth">
   <property name="text">
    <string>添加相机传感器参数</string>
   </property>
  </action>
  <action name="action_viewPLY">
   <property name="text">
    <string>打开PLY文件</string>
   </property>
  </action>
  <action name="action_reconstrctMesh">
   <property name="text">
    <string>三角网模型重建</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>模型纹理映射</string>
   </property>
  </action>
  <action name="actionopen_mvs_file">
   <property name="text">
    <string>打开...</string>
   </property>
  </action>
  <action name="action_fullauto">
   <property name="text">
    <string>全自动工程配置</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>mvsviewer</class>
   <extends>QWidget</extends>
   <header>mvsviewer.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <slots>
  <slot>btn_onclick()</slot>
 </slots>
</ui>
